### ISSUE TITLE
Jackett not starting after autoupgrade - Error System.BadImageFormatException: Could not load file or assembly 'Microsoft.AspNetCore.Mvc.Core

### ISSUE THREAD

**Issue by thejimp** [Jackett/Jackett#5641]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue. Duplicating issues makes it more difficult for everyone to follow. Your issue may have already been solved in the past as well.

Provide a description of the feature request or bug, the more details the better.

If you are experiencing an issue with a tracker, then:
1. Use your browser to check you can access the site directly, and if a login is required, check you can login and that you do not have any outstanding account issues.
2. If you haven't already, try upgrading to the latest version of Jackett, your issue may have already been resolved.
3. If it is still not working for you, then a **full enhanced log must be included**. Instructions for obtaining logs are here: https://github.com/Jackett/Jackett#troubleshooting

(At the bottom of your Jackett Dashboard page) **Jackett version**:  
(if you installed using Jackett.Binaries.Mono.tar.gz) **Mono version**:

-------------------------------
Raspberry Pi 4, Raspbian Buster
Hi, I've had a problem recently where Jackett stops and won't start again.  I think this may be related to an autoupdate.  I'm running as user 'vpn', and the /opt/Jackett folder is chowned by vpn.

Don't think it's the same as #3635
Here's my log.txt:
[log.txt](https://github.com/Jackett/Jackett/files/3378834/log.txt)

```
vpn@raspberrypi:/opt/Jackett $ ./jackett -t -l
07-10 18:53:03 Info Starting Jackett v0.11.475.0
07-10 18:53:03 Info Logging enabled.
07-10 18:53:03 Info Tracing enabled.
07-10 18:53:04 Debug Creating web host...
07-10 18:53:04 Debug Content root path is: /opt/Jackett/Content
07-10 18:53:05 Error System.BadImageFormatException: Could not load file or assembly 'Microsoft.AspNetCore.Mvc.Core, Version=2.2.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'. The module was expected to contain an assembly manifest.
File name: 'Microsoft.AspNetCore.Mvc.Core, Version=2.2.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'
   at Microsoft.AspNetCore.Hosting.ConventionBasedStartup.ConfigureServices(IServiceCollection services)
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.EnsureApplicationServices()
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.Initialize()
   at Microsoft.AspNetCore.Hosting.WebHostBuilder.Build()
   at Jackett.Server.Program.Main(String[] args)

 System.BadImageFormatException: Could not load file or assembly 'Microsoft.AspNetCore.Mvc.Core, Version=2.2.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'. The module was expected to contain an assembly manifest.
File name: 'Microsoft.AspNetCore.Mvc.Core, Version=2.2.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'
   at Microsoft.AspNetCore.Hosting.ConventionBasedStartup.ConfigureServices(IServiceCollection services)
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.EnsureApplicationServices()
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.Initialize()
   at Microsoft.AspNetCore.Hosting.WebHostBuilder.Build()
   at Jackett.Server.Program.Main(String[] args)



Unhandled Exception: System.BadImageFormatException: Could not load file or assembly 'Microsoft.AspNetCore.Mvc.Core, Version=2.2.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'. The module was expected to contain an assembly manifest.
   at Microsoft.AspNetCore.Hosting.ConventionBasedStartup.ConfigureServices(IServiceCollection services)
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.EnsureApplicationServices()
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.Initialize()
   at Microsoft.AspNetCore.Hosting.WebHostBuilder.Build()
   at Jackett.Server.Program.Main(String[] args)
Aborted

```

**Comment by thejimp** [comment]
Update log if this is useful:
[updater.txt](https://github.com/Jackett/Jackett/files/3379151/updater.txt)
Any help appreciated

**Comment by garfield69** [comment]
similar to #5208.
there appears to be an intermittent issue with the updater process that trips up on some systems.
until it is resolved, I suggest disabling the auto-update by using the Jackett dashboard **Disable auto update** checkbox and clicking on the **Apply Server Settings** button.
But first you will need to install manually from https://github.com/Jackett/Jackett/releases to restore service.

**Comment by thejimp** [comment]
Thanks - yes, does appear to be similar.  Previously I'd untarred the new release into the folder and it solved  the problem, but it's happened again so thought I'd log it as a bug.  I've installed it in a separate folder from the tarball and that appears to be running fine.  Will disable auto updates for the moment, but please advise if it's helpful for debugging for me to try auto-updating again.  Many thanks.

**Comment by ngosang** [comment]
@thejimp please, test this fix => https://github.com/Jackett/Jackett/issues/5208#issuecomment-547709864

**Comment by thejimp** [comment]
> @thejimp please, test this fix => [#5208 (comment)](https://github.com/Jackett/Jackett/issues/5208#issuecomment-547709864)

Perfect.
Manually updated to 0.12.898 and then auto updated to 0.12.907.0 with no problems. Thanks
